cmd/tts.go

```diff
@@ -88,5 +88,5 @@ var ttsCmd = &cobra.Command{
 func init() {
 	rootCmd.AddCommand(ttsCmd)
 	ttsCmd.Flags().String("google-service-account", "", "google service account JSON file")
-	ttsCmd.Flags().String("language-code", "en-US", "text-to-speech Language Code (de-DE, ja-JP,...)")
+	ttsCmd.Flags().String("voice-name", "en-US", "text-to-speech Voice (en-US-Wavenet-G, pl-PL-Wavenet-A, pl-PL-Wavenet-B, de-DE-Wavenet-A)")
```

Comment:
Can we also change the default from `enUS` to one of the valid names?

Reply:
I have no way to test it actually, since I can't generate arm7 to test it on, so it's pure theory craft. I will make some changes then.

Reply:
I created new commit for that.